src/store/storeConfig.ts

```diff
@@ -85,6 +97,12 @@ interface BareRootState {
 const initialState: BareRootState = {
   menu: null,
   globalLoading: true,
+  defaultVisibility: {
+    appError: true,
+    notificationManager: true,
+    onboardingNotification: true,
+    breadcrumbs: true,
+  },
```

Comment:
Are these always `true` in this environment?

Reply:
Yes.

Reply:
What would "something different" be? Currently, this is only related to the strictly boolean question of whether something shows up in the UI.